plugins/meson.build

```diff
@@ -1,149 +1,66 @@
 subdir('acpi-dmar')
 subdir('acpi-facp')
+subdir('altos')
+subdir('amt')
+subdir('ata')
 subdir('bcm57xx')
+subdir('bios')
+subdir('ccgx')
+subdir('colorhug')
 subdir('cpu')
+subdir('cros-ec')
+subdir('dell')
+subdir('dell-dock')
+subdir('dell-esrt')
+subdir('dfu')
+subdir('dfu-csr')
+subdir('ebitdo')
+subdir('elantp')
+subdir('emmc')
 subdir('ep963x')
+subdir('fastboot')
+subdir('flashrom')
+subdir('fresco-pd')
+subdir('goodix-moc')
+subdir('hailuck')
 subdir('iommu')
+subdir('jabra')
 subdir('linux-lockdown')
 subdir('linux-sleep')
```

Comment:
I still think that we should make it clear that a few plugins like `linux-lockdown` and `linux-sleep` are truly linux only like my original PR had.

Reply:
Ohh, I missed that bit. Got a list offhand of the ones you worked out were linux specific?

Reply:
These were the ones I made out, feel free to disagree or add anything else you think though.

```
  subdir('acpi-dmar')       #/sys/firmware/acpi
  subdir('acpi-facp')       #/sys/firmware/acpi
  subdir('iommu')           #linux kernel iommu
  subdir('linux-lockdown')
  subdir('linux-sleep')
  subdir('linux-swap')
  subdir('linux-tainted')
  subdir('pci-bcr')         #pci
  subdir('pci-mei')         #pci
  subdir('upower')
  subdir('pixart-rf')       #linux kernel hidraw
```

Reply:
Do we want to run acpi-facp-self-test on !linux? I'm erring on yes.